docs/content/manual/release-specific/v1.4.0/test-system-backup.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Test Longhorn system backup
+---
+
+## Related issue
+https://github.com/longhorn/longhorn-manager/pull/1593
```

Comment:
I think, we don't need the PR link in the related issue.

Reply:
Removed.